inspirehep/dojson/hep/model.py

```diff
@@ -26,4 +26,19 @@
 from dojson import utils
 
 hep = Overdo()
+
+
+@hep.over('book', '^773..')
```

Comment:
I guess this someone is me :P :D
On Dec 15, 2015 6:24 PM, "Javier Martin Montull" <notifications@github.com>
wrote:

> In inspirehep/dojson/hep/model.py
> <https://github.com/inspirehep/inspire-next/pull/636#discussion_r47667895>
> :
>
> > @@ -26,4 +26,19 @@
> >  from dojson import utils
> >
> >  hep = Overdo()
> > +
> > +
> > +@hep.over('book', '^773..')
>
> Good point @jalavik <https://github.com/jalavik> Actually did not read
> this comment in time and merged #666
> <https://github.com/inspirehep/inspire-next/pull/666> after a slight
> modification that allows to keep using hep.do
>
> The effect is the same, so at least somebody can learn how to inject a
> function into a class instance :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inspirehep/inspire-next/pull/636/files#r47667895>.
>